src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const basicDiscount = 20;
+  const extraDiscount = 50;
+  const rentCosts = 40;
+
+  let total = days * rentCosts;
+
+  if (days >= 3) {
+    total = (days * rentCosts) - basicDiscount;
+  };
+
+  if (days >= 7) {
+    total = (days * rentCosts) - extraDiscount;
+  };
```

Comment:
You should return the value immediately in `if` statements without overwriting the variable.

```
if (condition) {
    return x;
};
```